### PR TITLE
Update 0074-jettons-standard.md

### PR DESCRIPTION
**Pull request by awesome-doge**
Fix broken links

### CODE REVIEW

**Comment by titarenko88** [comment]
St

**Comment by hosenhsimi** [comment]
9c8bde6290dc610568b8024c989d83e1e2680381your-project/
    theme_common/
    theme_override/
    content/

**Review by hosenhsimi — APPROVED** [review]

**Review by Elgayar777 — APPROVED** [review]
- [ ]

**Comment by Shojashakib** [comment]
A

**Review by Elgayar777 — APPROVED** [review]
- [ ]

File: text/0074-jettons-standard.md
Comment (dwcoen1234) [inline text/0074-jettons-standard.md:1]: text/0074-jettons-standard.md

**Pull request merged**